server/odc-service/src/main/java/com/oceanbase/odc/service/connection/logicaldatabase/core/executor/execution/thread/ExecutorServiceManager.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.connection.logicaldatabase.core.executor.execution.thread;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import lombok.Getter;
+
+/**
+ * @Author: Lebie
+ * @Date: 2024/8/26 21:58
+ * @Description: []
+ */
+@Getter
```

Comment:
seems ThreadPoolFactory ?

may move to common util

Reply:
Actually it's about `ExecutorService`, it manages a lifecycle of a singleton `ExecutorService`.
It may be not ready for common use, so I prefer to keep this class in `logicaldatabase` package.